playbook/app/pb_kits/playbook/pb_table/_table.jsx

```diff
@@ -57,9 +61,10 @@ const Table = (props: TableProps) => {
         {
           'table-card': container,
           'table-dark': dark,
-          'data_table': dataTable,
+          data_table: dataTable,
```

Comment:
Is there a reason why `''` are removed ?

Reply:
unsure why they were removed, i readded them and nothing seems to have broken. I remember when i was working with jasper on this, we came across some linting errors. So im going to assume that was the root cause of us removing them.

Reply:
I think we should update the code only if there's a purpose behind. If there's no reason behind change then we need to confirm it with  @jasperfurniss .